pkgs/development/tools/rust/cargo-make/default.nix

```diff
@@ -4,31 +4,21 @@ rustPlatform.buildRustPackage rec {
   pname = "cargo-make";
   version = "0.27.0";
 
-  src =
-    let
-      source = fetchFromGitHub {
-        owner = "sagiegurari";
-        repo = pname;
-        rev = version;
-        sha256 = "197r7rlgk8csvq612gz4431vvrvyxhcmfswarsdczp4b94pzaa6h";
-      };
-    in
-    runCommand "cargo-make-src" {} ''
-      cp -R ${source} $out
-      chmod +w $out
-      cp ${./Cargo.lock} $out/Cargo.lock
-    '';
```

Comment:
This was really clever . . . I've now replaced it with the cookie cutter implementation though :)

Since it was effectively spoofing the src attr as-if it had a Cargo.lock, it would have worked.